conformance_pack/guardduty.sp

```diff
@@ -77,6 +87,37 @@ query "guardduty_enabled" {
   EOQ
 }
 
+query "guardduty_no_high_severity_findings" {
+  sql = <<-EOQ
+    with finding_count as (
+      select
+        detector_id,
+        count(*) as count
+      from
+        aws_guardduty_finding
+      group by
+        detector_id
+    )
+    select
+      arn as resource,
+      case
+        when fc.count = 0 or fc.count is NULL then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when fc.count = 0  or fc.count is NULL then d.detector_id || ' is enabled and does not have high severity findings.'
+        else d.detector_id || ' is enabled and has ' || fc.count ||' high severity findings.'
+      end as reason
+      ${replace(local.tag_dimensions_qualifier_sql, "__QUALIFIER__", "d.")}
+      ${replace(local.common_dimensions_qualifier_sql, "__QUALIFIER__", "d.")}
+    from
+      aws_guardduty_detector as d
+      left join finding_count as fc on fc.detector_id = d.detector_id
+    where 
+      status = 'ENABLED';
```

Comment:
can we remove this where clause and add one condition to skip the control if `status <>  'ENABLED`?